src/Core/src/Platform/iOS/MauiTextView.cs

```diff
@@ -161,5 +173,18 @@ void ShouldCenterVertically()
 				_ => new CGPoint(0, 0),
 			};
 		}
+
+		void UpdatePlaceholderFontSize(UIFont? value)
```

Comment:
What happens if you set the label font to null? What does the text view do?

I am wondering if we always set label.Font = textView.Font, will that be an issue? Looking at the code path, if I set font size to be 30 and then 20, and then go null, the textView may do something (say revert back to OS defaults - not sure) but the _defaultPlaceholderSize will be 30 and might be a mismatch.

Reply:
did you check my last update, _defaultPlaceholderSize will only be updated the 1st time.